src/renderers/dom/shared/ReactDOMComponent.js

```diff
@@ -322,6 +322,49 @@ function trapBubbledEventsLocal() {
         ),
       ];
       break;
+    case 'video':
+    case 'audio':
+      var mediaEvents = {
```

Comment:
You mean I should just copy the code for audio and video? What's the benefit in that?